.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 8
+script:
+  - node_modules/bs-platform/bin/bsb -make-world
```

Comment:
Uh, ignore the SCSS part, Tea does not have any, ignore that...  What about the `npm install` part then?  ^.^;